source/fundamentals/builders/filters.txt

```diff
@@ -11,3 +11,684 @@ Filters Builders
+      collection.find(equalComparison).into(results);
+      for (Document result : results) {
+         System.out.println(result.toJson());
+      }
+   
+   The Bson renders as:
+
+   .. code-block:: none
+      :copyable: false
+   
+      {
+         "qty": 20
+      }
+
+   The output of the above code snippet should look something like this:
+
+   .. code-block:: json
+      :copyable: false
+   
+      {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+      {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+
+.. example::
+
+   The following example creates a filter that selects all documents in
+   the paint_purchaces collection because the predicate is empty: 
+
+   .. code-block:: java
+      
+      import static com.mongodb.client.model.Filters.empty
+
+      // <MongoCollection setup code here>
+
+      Bson emptyComparison = empty();
+      List<Document> results = new ArrayList<>();
+      collection.find(emptyComparison).into(results);
+      for (Document result : results) {
+         System.out.println(result.toJson());
+      }
+   The Bson renders as:
+
+   .. code-block:: none
+      :copyable: false
+   
+      { }
+
+   The output of the above code snippet should look something like this:
+   
+   .. code-block:: json
+      :copyable: false
+      
+      {"_id": 1, "color": "red", "qty": 5, "vendor": ["A"]}
+      {"_id": 2, "color": "purple", "qty": 10, "vendor": ["C", "D"]}
+      {"_id": 3, "color": "blue", "qty": 8, "vendor": ["B", "A"]}
+      {"_id": 4, "color": "white", "qty": 6, "vendor": ["D"]}
+      {"_id": 5, "color": "yellow", "qty": 11, "vendor": ["A", "B"]}
+      {"_id": 6, "color": "pink", "qty": 5, "vendor": ["C"]}
+      {"_id": 7, "color": "green", "qty": 8,"vendor": ["B", "C"]}
+      {"_id": 8, "color": "orange", "qty": 7, "vendor": ["A", "D"]}
+
+.. _logical:
+
+Logical
+-------
+
+The logical operator methods include:
+
+.. list-table::
+   :header-rows: 1
+   :stub-columns: 1
+
+   * - Logical Operator
+     - 
+      
+   * - ``and()``
+     - Joins filters with a logical AND and selects documents that match the conditions of both filters.
+
+   * - ``or()``
+     - Joins filters with a logical OR and selects documents that match the conditions of either filters.
+      
+   * - ``not()``
+     - Inverts the effect of a query expression and selects documents that do not match the filter.
+     
+   * - ``nor()``
+     - Joins filters with a logical NOR and selects documents that fail to match both filters.
+
+The following code snippet shows how to use a logical filter:
+
+.. code-block:: java
+   
+   import static com.mongodb.client.model.Filters.*;
+
+   // <MongoCollection setup code here>
+
+   collection.find(<logical filter>);
+
+.. example::
+
+   The following example creates a filter that selects documents where
+   the value of the ``qty`` field is greater than ``8`` or the value
+   of the ``color`` field equals ``"pink"`` in the paint_purchaces collection: 
```

Comment:
For values, let's try to stick with using quotes and no monospace. E.g. "8" and "pink".
Also, remember to keep monospace formatting on collection names.